src/Components/Shared/PlayerButton.tsx

```diff
@@ -2,10 +2,17 @@ import React from "react"
 import { Button } from "react-bootstrap"
 import { TrackSymbols } from "../../Constants/Symbols"
 import * as PlayerEndpoints from "../../Endpoints/Player"
+import { AuthenticationContext } from "../../Models/Authentication"
+import { Spotify } from "../../Models/SpotifyObjects/PlayerObjects"
 
+/* eslint-disable */
```

Comment:
More linting issues? This is being used below so I'd assume it's not the same issue as last time?